### ISSUE TITLE
Additional proxy settings won't be persisted in server.xml.erb

### ISSUE THREAD

**Issue by pdirksen** [brycejohnson/puppet-jira#43]
Something is wrong in ```server.xml.erb```:

```
<% if @proxy -%>
  <% @proxy.sort.each do |key,value| -%>
    <%= key %> = <%= "\'#{value}\'" %>
  <% end -%>
<% end -%>
```
Despite setting the parameter 'proxy' it will not be persisted in server.xml

**Comment by mkrakowitzer** [comment]
I am not able to reproduce this issue. May I ask what version of puppet are you using? What OS are you running this module on?

**Comment by mkrakowitzer** [comment]
@pdirksen have you managed to resolve this issue? or have any additional information that can help me trouble shoot the problem you are having. 

If not, @brycejohnson I am going to close this issue.

**Comment by mkrakowitzer** [comment]
@pdirksen Could you please let me know if this is possibly resolved? I recently added future parser to our testing and I could reproduce this issue. It should be fixed in 1.2.0

**Comment by pdirksen** [comment]
It works now.
Thanks
